pylele_tube.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+"""
+    Tube Solid
+"""
+
+from pylele_solid import LeleSolid
+from pylele_api import Shape
+
+class Tube(LeleSolid):
```

Comment:
Great addition! Another would be RndRod since its just cylinder with spheres on both end.